src/mozillavpn.h

```diff
@@ -362,6 +363,9 @@ class MozillaVPN final : public QObject {
 
   void maybeRegenerateDeviceKey();
 
+  QList<Server> filterServerList(const QList<Server>& servers) const;
+  void checkServerCooldownTimeouts();
```

Comment:
Where do we use this?

Reply:
Thank you! I forgot to remove that while refactoring.